### ISSUE TITLE
Option to flip Watson flag?

### ISSUE THREAD

**Issue by ETalanian** [geheur/watson-master-clue-tracker#7]
Hello,

Love this plugin, could you please add an option so that the little W on clues only appear if Watson _needs_ the clue, instead of if he already has it?

Thanks <3

**Comment by geheur** [comment]
Could be done, I always forget what that icon means anyways. Maybe I should change it to a red x or green checkmark based on if he doesn't or does have it.

**Comment by DelVision** [comment]
Still hoping for reverse option to only have the "w" on the clues that Watson needs instead of the ones that he already has

**Comment by geheur** [comment]
Yeah it would be good. It's kind of hard to just simply reverse this because it would be confusing to people who are used to it. Even a toggle would be weird - imagine two people discuss this plugin and one has the reversed indications but forgot that one person enabled that toggle. That woudl be confusing.

It might be doable if I could have an icon that had a "filled in" and "not filled in" version. That way it could be "filled in" when watson has the clue and not filled in when you should give it to him. That would be clear, and if there was an option to disable one of the two icons it would still be clear.

I'm not sure what such an icon would be. I'm not an artist and I'm not sure what icon represents watson well.

**Comment by geheur** [comment]
I think it could be nice to have 4 dots, either filled or unfilled, on each clue scroll to let you know what watson needs. This is easy to show whether he does or doesn't have them (filled or unfilled dots), and it also shows what clues you have left to get.

Then I could make the current thing display in gray when watson doesn't have it, and let people toggle it off for when he does have it, if they want.

**Comment by DelVision** [comment]
I don't know how it would be confusing for people who choose to change the options for their personal preferences and adding extra icons would just make it worse..

Simply put, the most clear way to show that watson needs an clue would be to show that little "w" on the clue that's already there, and if he already has one then not show it..

You don't need information about clues that you have already delivered to watson, that's just pointless information and if you need to know what clues watson is missing, that's already there when you mouse over any clue.

Basically the plugin is a great reminder, but the icon is working logically backwards that you don't get notified when he needs a clue and you get notified when he already has one when you don't need to know that information..

**Comment by geheur** [comment]
extra icons was an alternative option that I think would be cool. I agree that the existing thing works backwards atm. I just want a different icon for "watson has it" vs "watson doesn't have it".

**Comment by geheur** [comment]
I added options for this in 1.2.0. Hope it's satisfactory <3.
